tests/test_feature_extraction.py

```
@@ -108,11 +112,19 @@ def test_radiomicFeatureExtraction(nsclcMetadataPath):
         "Volume feature is incorrect"
 
 
-def test_radiomicFeatureExtraction_output(nsclcMetadataPath):
-    """Test output creation from radiomic feature extraction"""
+def test_NSCLC_radiomicFeatureExtraction_output(nsclcMetadataPath):
+    """Test output creation from radiomic feature extraction for SEG dataset"""
     actual = radiomicFeatureExtraction(nsclcMetadataPath,
```

Ruff (0.8.2)

Local variable `actual` is assigned to but never used

Remove assignment to unused variable `actual`

(F841)